src/ontology/uberon-edit.obo

```diff
@@ -224996,7 +224993,6 @@ xref: RO:0002002
 id: has_branching_part
 name: has branching part
 xref: RO:0002569
-is_transitive: true
```

Comment:
From https://github.com/obophenotype/uberon/issues/2453#issuecomment-1118435290:

> [Typedef]
> id: has_branching_part
> name: has branching part
> xref: RO:0002569
> is_transitive: true
> Looks reasonable, although potential for massive proliferation of trivial inferences.
> 
>  Add to RO https://github.com/oborel/obo-relations/pull/612

Do not see instruction to delete this.